Description: comment out dependencies specific to windows and macos
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
Last-Update: 2025-09-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -57,20 +57,20 @@
 [dependencies.serde_json]
 version = "~1.0"
 
-[target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.core-foundation]
-version = "~0.10"
+# [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.core-foundation]
+# version = "~0.10"
 
-[target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.core-foundation-sys]
-version = "~0.8"
+# [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.core-foundation-sys]
+# version = "~0.8"
 
-[target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.io-kit-sys]
-version = "~0.4"
+# [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.io-kit-sys]
+# version = "~0.4"
 
-[target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.libc]
-version = "~0.2"
+# [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.libc]
+# version = "~0.2"
 
-[target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.mach2]
-version = "~0.4"
+# [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.mach2]
+# version = "~0.4"
 
-[target."cfg(windows)".dependencies.libc]
-version = "0.2"
+# [target."cfg(windows)".dependencies.libc]
+# version = "0.2"
